csrankings-p.csv

```diff
@@ -122,7 +122,7 @@ Paola Quaglia,University of Trento,https://webapps.unitn.it/du/en/Persona/PER000
 Paola Velardi,Sapienza University of Rome,https://sites.google.com/a/di.uniroma1.it/paolavelardi,yf0g6zkAAAAJ
 Paolo A. G. Sivilotti,Ohio State University,http://web.cse.ohio-state.edu/~paolo,NOSCHOLARPAGE
 Paolo Baldan,University of Padova,https://www.math.unipd.it/~baldan,mPd-c8YAAAAJ
-Paolo Bientinesi,RWTH Aachen,https://www.aices.rwth-aachen.de/people/bientinesi,G6Z5vx8AAAAJ
+Paolo Bientinesi,RWTH Aachen University,https://www.aices.rwth-aachen.de/people/bientinesi,G6Z5vx8AAAAJ

```

Comment:
Has a dual affiliation with Umea (https://hpac.cs.umu.se/~pauldj/) - please confirm that the RWTH appointment is >= 75% time.